controllers/config/constants.go

```diff
@@ -15,6 +15,25 @@ const (
 	ENVNodeName    = "NODE_NAME"
 	ENVKubeVersion = "KUBE_VERSION"
 
+	//  Default images for containers
```

Comment:
**Query:**  Are default images constant? Based on tags it seems they are likely to change with version/tag change. So are we planning to update default image under this file when ever  new image is build with new tags ?

Reply:
The default images here will be overwritten if the images are provided in CR. But I think most of the customers would use the default configuration (where they don't need to know the image names and tags), so yes, we will have to update the images here whenever there is change.